src/templates/post/index.jsx

```diff
@@ -84,9 +84,6 @@ export const pageQuery = graphql`
         author {
           id
           name
-          bio
-          twitter
-            email
```

Comment:
Tady jsme odstranili i pole e-mail, které používáme – nejspíš tě zmátlo to špatné odsazení. Můžem ho prosím vrátit?

Reply:
👍 Vráceno se správným odsazením